storage/remote/queue_manager.go

```diff
@@ -166,79 +177,92 @@ func (t *StorageQueueManager) Describe(ch chan<- *prometheus.Desc) {
 	ch <- t.queueCapacity.Desc()
 }
 
+// QueueLength returns the number of outstanding samples in the queue.
+func (t *StorageQueueManager) QueueLength() int {
```

Comment:
there is a field called `queueLength` which clashed with this.  Any suggestions for a different name?